src/ahocorapy/keywordtree.py

```diff
@@ -112,13 +112,14 @@ def search_all(self, text):
                              ' No search allowed. Call finalize() first.')
         if self._case_insensitive:
             text = text.lower()
-        current_state = self._zero_state
+        zero_state = self._zero_state
+        current_state = zero_state
         for idx, symbol in enumerate(text):
             current_state = current_state.transitions.get(
-                symbol, self._zero_state.transitions.get(symbol,
-                                                         self._zero_state))
+                symbol, zero_state.transitions.get(symbol,
+                                                         zero_state))
```

Comment:
Please fix the formatting to be PEP8 compliant as checked by the CI.

Reply:
Done. Thanks in advance for the release :)